test/Microsoft.Azure.Devices.Edge.Test/Microsoft.Azure.Devices.Edge.Test.csproj

```diff
@@ -29,6 +29,7 @@
     <ProjectReference Include="..\..\edge-util\src\Microsoft.Azure.Devices.Edge.Util\Microsoft.Azure.Devices.Edge.Util.csproj" />
     <ProjectReference Include="..\..\edge-util\test\Microsoft.Azure.Devices.Edge.Util.Test.Common\Microsoft.Azure.Devices.Edge.Util.Test.Common.csproj" />
     <ProjectReference Include="..\Microsoft.Azure.Devices.Edge.Test.Common\Microsoft.Azure.Devices.Edge.Test.Common.csproj" />
+    <ProjectReference Include="..\modules\ModuleLib\Microsoft.Azure.Devices.Edge.ModuleUtil.csproj" />
```

Comment:
I don't see where `ModuleUtils` is being used in the test project. Can it be removed?

Reply:
It's being used for TestConstants.PriorityQueues.Default as a way to have the source of truth for the "Default" keyword be shared across modules (e.g. this framework + loadGen module). If we continue bringing in other modules, ModuleUtil is a good place to put common logic like this

Reply:
Nvm